CONTRIBUTING-PYTHON.md

```diff
@@ -221,3 +227,7 @@ make lint
 # Fix linting issues
 make lint-fix
 ```
+
+## Changelog
+
+Currently the CHANGELOG.md is managed manually. Please include a changelog entry in the appropriate package's CHANGELOG.md file when you add a new feature or fix a bug.
```

Comment:
There will be a fast-follow to introduce a Python-specific changelog management tool in a separate PR. That PR will update these instructions